assets/block-editor/data/messages-store/index.jsx

```diff
@@ -0,0 +1,24 @@
+/**
+ * WordPress dependencies.
+ */
+import { registerStore } from '@wordpress/data';
+
+/**
+ * Internal dependencies.
+ */
+import { MESSAGES_STORE } from './name';
+import * as actions from './actions';
+import reducer from './reducer';
+import * as selectors from './selectors';
+import * as resolvers from './resolvers';
+
+// Register the Sensei Messages store.
+registerStore( MESSAGES_STORE, {
```

Comment:
I don't think we want to create a store just for messages. We'd then have to create separate stores for courses, lessons, questions etc. The store should probably be generic enough to hold all Sensei state. 

Calypso uses [a single store for its state](third-era-redux-global-state-tree-december-2015---present). [WooCommerce uses three separate stores](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/master/assets/js/data/README.md).

We may benefit from spending some additional time researching and thinking about how to best design the data store.